src/azure-cli/azure/cli/command_modules/containerapp/custom.py

```diff
@@ -4803,7 +4805,7 @@ def update_openid_connect_provider_settings(cmd, resource_group_name, name, prov
     if client_secret is not None:
         final_client_secret_setting_name = get_oidc_client_setting_app_setting_name(provider_name)
         registration["clientSecretSettingName"] = final_client_secret_setting_name
-        set_secrets(cmd, name, resource_group_name, secrets=[f"{final_client_secret_setting_name}={client_secret}"], no_wait=True, disable_max_length=True)
+        set_secrets(cmd, name, resource_group_name, secrets=[f"{final_client_secret_setting_name}={client_secret}"], no_wait=False, disable_max_length=True)
```

Comment:
why change no wait value here?

Reply:
When set_secrets during update settings, need to wait until it finish, otherwise it will throw error about secret not found in ocntainerapp, for example:

```
src\azure-cli\azure\cli\command_modules\containerapp\custom.py:4214: in update_aad_settings
   handle_raw_exception(e)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _  

e = HTTPError('Bad Request({"error":{"code":"AuthConfigSecretRefNotFound","message":"Secret with name \'microsoft-provider-authentication-secret\' is not found in configuration of container app \'containerapp-auth4chpvat\'."}})')
```